### PR TITLE
Add dcache dcap

### PR DESCRIPTION
**Pull request by chrisburr**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/dcap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - {{ compiler('cxx') }}
+    - automake
+    - make
+    - pkg-config
+    - libtool
+    - sed
+  host:
+    - gct
+    - krb5
+    - openssl 1.0.2r
```
Comment (chrisburr) [inline recipes/dcap/meta.yaml]: I'm using openssl 1.0 as I'm building something which needs it so I'll migrate over to 1.1 in the feedstock.

I've checked and `run_export` is used correctly in `openssl=1.0.2r`.

**Comment by chrisburr** [comment]
@conda-forge/staged-recipes This is ready for review

File: recipes/dcap/build.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    --with-globus-lib="${PREFIX}/lib" \
+    --with-globus-include="${PREFIX}/include" \
+    --with-krb5-gssapi-include="${PREFIX}/include"
+
+if [ "$(uname)" == "Darwin" ]; then
+    sed -i 's/notelnet="0"/notelnet="1"/g' configure
+fi
+
+make -j${CPU_COUNT}
+make install
```
Comment (ocefpaf) [inline recipes/dcap/build.sh]: Is there a `make check`?

File: recipes/dcap/build.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+
+sh bootstrap.sh
+
+declare -a PLATFORM_FLAGS
+if [ "$(uname)" == "Linux" ]; then
+    PLATFORM_FLAGS+=(--with-sysroot="${PREFIX}")
```
Comment (isuruf) [inline recipes/dcap/build.sh]: PREFIX is not the sysroot

Comment (chrisburr) [reply recipes/dcap/build.sh]: It's not even needed so I've removed it.

File: recipes/dcap/build.sh (hunk 2 above)
Comment (chrisburr) [reply recipes/dcap/build.sh]: Yes there is 👍

**Comment by scopatz** [comment]
Thanks @chrisburr !

**Comment by chrisburr** [comment]
Thanks everyone! 🎆

**Pull request merged**